classes/PodsInit.php

```diff
@@ -346,8 +346,9 @@ public function register_assets() {
 		if ( is_admin() ) {
 			$screen = get_current_screen();
 
-			// DFV must be enqueued on the media library page for items in grid mode (see #4785)
-			if ( $screen->base && 'upload' === $screen->base ) {
+			// DFV must be enqueued on the media library page for items in grid mode (#4785)
+			// and for posts due to the possibility that post-thumbnails are enabled (#4945)
+			if ( $screen->base && in_array( $screen->base, array( 'upload', 'post') ) ) {
```

Comment:
@pglewis I've had to add 'customize' to the list, so that it works in the theme customization page too (`/wp-admin/customize.php`).

The line becomes:
```
if ( $screen->base && in_array( $screen->base, array( 'upload', 'post', 'customize' ) ) ) {
```